[Bug] GradleException – Cannot find a version of 'androidx.test:core' that satisfies the version constraints

Master (4485b7f6475218149fdd0eb3f4a676849005f513) (geckoNightlyDebug)

On attempt to run UI tests:

```Caused by: org.gradle.internal.resolve.ModuleVersionResolveException: Could not resolve androidx.test:core:1.2.1-alpha02.```

```Caused by: org.gradle.api.GradleException: Cannot find a version of 'androidx.test:core' that satisfies the version constraints: 
   Dependency path 'fenix:app:unspecified' --> 'androidx.test:core:1.2.1-alpha02'
   Constraint path 'fenix:app:unspecified' --> 'androidx.test:core:{strictly 1.1.0}' because of the following reason: geckoNightlyDebugRuntimeClasspath uses version 1.1.0
   Dependency path 'fenix:app:unspecified' --> 'androidx.fragment:fragment-testing:1.2.0-alpha02' --> 'androidx.test:core:1.1.0'```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -452,8 +452,9 @@ dependencies {
 
     androidTestImplementation Deps.androidx_test_core
     androidTestImplementation Deps.espresso_idling_resources
-    debugImplementation Deps.androidx_fragment_testing
-
+    debugImplementation (Deps.androidx_fragment_testing) {
+        exclude group: 'androidx.test', module: 'core'
+    }
 
     androidTestImplementation Deps.tools_test_runner
     androidTestImplementation Deps.tools_test_rules

